lib/rules/comma-dangle.js

```diff
@@ -128,10 +132,14 @@ module.exports = function(context) {
         var sourceCode = context.getSourceCode();
         var trailingToken = sourceCode.getTokenAfter(lastItem);
         if (trailingToken.value !== ",") {
-            context.report(
-                lastItem,
-                lastItem.loc.end,
-                MISSING_MESSAGE);
+            context.report({
+                node: lastItem,
+                loc: lastItem.loc.end,
+                message: MISSING_MESSAGE,
+                fix: function(fixer) {
+                    return fixer.insertTextAfter(lastItem, ",");
```

Comment:
This is precisely why I said we need to implement small numbers of autofixing rules first - we're going to run into collisions, and right now, we have no way to fix them.

The problem in this case is that you'll add a comma and then comma-spacing won't be able to validate the spacing around it, which could mean there's a hidden warning that won't be shown until you run ESLint again. That's not a situation we want to be.

`eqeqeq` doesn't have this problem because it's only adding an extra character into an operator, so all other rules related to spacing and such are still valid.

`semi` could potentially have a problem.

`*-spacing` should mostly be safe as no two rules should checking the same space between tokens.